src/routes/Issues/components/Issues.tsx

```diff
@@ -63,13 +63,21 @@ const getParamsFromProps = (props: IIssuesProps): IParams => {
         ? [language]
         : language
       : undefined,
-    type: type ? (typeof type === "string" ? [type] : type) : undefined,
+    type: type 
+      ? typeof type === "string" 
+        ? [type] 
+        : type 
+      : undefined,
     project_id: project_id
       ? typeof project_id === "string"
         ? [project_id]
         : project_id
       : undefined,
-    ordering
+    ordering: ordering
+      ? typeof ordering === "string"
```

Comment:
This check should check if it’s an array and only then return the first element

Reply:
The below lines return first element of an array if it is of the type string[]